Flow.Launcher.Plugin/SharedCommands/FilesFolders.cs

```diff
@@ -1,7 +1,9 @@
 ﻿using System;
 using System.Diagnostics;
 using System.IO;
+#if !DEBUG
```

Comment:
why this?

Reply:
It was to neutralize warning. `System.Windows` is only used for release. Just realized I should neutralize the warning instead.